.github/stale.yml

```diff
@@ -0,0 +1,56 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# General configuration
+
+# Pull request specific configuration
+pulls:
+  staleLabel: awaiting changes
+  # Number of days of inactivity before an Issue or Pull Request becomes stale
+  daysUntilStale: 45
+  # Number of days of inactivity before a stale Issue or Pull Request is closed.
+  # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+  daysUntilClose: 30
+  # Comment to post when marking as stale. Set to `false` to disable
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    activity in the last 6 weeks. It will be closed in 30 days if no further activity occurs. 
```

Comment:
6 weeks != 45 days

Reply:
It's 42, but is it that important?

Reply:
You know, it bothered me when I first reviewed the PR, but now it doesn't. :man_shrugging:

Reply:
I agree with @noroadsleft, we say 30/90 days for issues, but 30 days/ 6 weeks for PRs. Having it be correct will avoid confusion later.

Reply:
Okay, will update then!

Reply:
Okay, text updated to use the days